bitbot/nick.go

```diff
@@ -22,3 +22,16 @@ var NickTakenTrigger = NamedTrigger{ //nolint:gochecknoglobals,golint
 		return false
 	},
 }
+
+var NickRecoverTrigger = NamedTrigger{ //nolint:gochecknoglobals,golint
+	ID:   "nick-recover",
+	Help: "Watch for QUIT messages, and recover nick at first occasion",
+	Condition: func(irc *hbot.Bot, m *hbot.Message) bool {
+		return m.Command == "QUIT" && m.From == b.Config.Nick
```

Comment:
This works if you're in the same channel but not if you don't share any channels with the conflicting nickname. I think it works for now and we can improve on it later.

Reply:
One alternative could be to have a goroutine periodically attempting nick takeover, but I don't know how I feel about adding a routine to something so simple.
We could add a `NickTakeOver` trigger too.